clang/lib/Basic/Targets.cpp

```diff
@@ -117,6 +118,13 @@ std::unique_ptr<TargetInfo> AllocateTarget(const llvm::Triple &Triple,
   default:
     return nullptr;
 
+  case llvm::Triple::UnknownArch:
+    /* native_cpu is only known to Clang, not to LLVM. */
```

Comment:
Please use C++-style comments.
```suggestion
    // native_cpu is only known to Clang, not to LLVM.
```